tests/test_ensembles/conftest.py

```diff
@@ -106,6 +114,14 @@ def voting_ensemble_pipeline(
     return pipeline
 
 
+@pytest.fixture
+def voting_ensemble_pipeline_int_timestamp(
```

Comment:
Why we use "int_timestamp" part in naming? It seems, that this pipeline can work with normal timestamps too.

Reply:
Yes, it can. But I created it exactly to work in integer timestamp. It replaces prophet with sarimax in `voting_ensemble_pipeline`.